pkg/console/subresource/deployment/deployment_test.go

```diff
@@ -412,6 +412,41 @@ func TestDefaultDownloadsDeployment(t *testing.T) {
 	infrastructureConfigHighlyAvailable := infrastructureConfigWithTopology(configv1.HighlyAvailableTopologyMode)
 	infrastructureConfigSingleReplica := infrastructureConfigWithTopology(configv1.SingleReplicaTopologyMode)
 
+	downloadsDeploymentPodSpec := corev1.PodSpec{
```

Comment:
Lets add the `SingleReplica` suffix, to make its clear that the Spec is single replica specific
```suggestion
	downloadsDeploymentPodSpecSingleReplica := corev1.PodSpec{
```

Reply:
@jhadvig good catch! Done.